src/modules/menu/components/advancedTools/ImportCatalogue/ImportCatalogue.vue

```diff
@@ -39,4 +48,13 @@ function onClear() {
     </div>
 </template>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import 'src/scss/media-query.mixin';
+@import 'src/scss/variables';
+
+.import-catalogue {
+    &.desktop-mode {
+        max-width: calc($menu-tray-width - 32px);
```

Comment:
that might be good, for future you or me, to explain that this 32px is some buffer so that the scroll bar won't change the size of the input

Reply:
I added a comment